### PR TITLE
Add new disposition workflow transition refuse.

### LINKED ISSUE #2346: Add new transition `ablehnen` to disposition workflow
Add a new exit-transition `ablehnen` to the states `disposition-state-in-progress` and `disposition-state-appraised`. The transition should put the disposition object in the `disposition-state-in-progress` state.

More information: https://basecamp.com/2768704/projects/12554551/todos/274416836

### PR DESCRIPTION
**Pull request by phgross**
The transition is only available for managers and archivists and in the `in progress` or disposed state.

Closes #2346 
Styling: https://github.com/4teamwork/plonetheme.teamraum/pull/487

@lukasgraf or @deiferni

### CODE REVIEW

File: opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <guard>
+      <guard-role>Records Manager</guard-role>
+      <guard-role>Archivist</guard-role>
+      <guard-role>Manager</guard-role>
+    </guard>
+  </transition>
+  <transition transition_id="disposition-transition-dispose" title="" new_state="disposition-state-disposed" trigger="USER" before_script="" after_script="">
+    <action url="%(content_url)s/content_status_modify?workflow_action=disposition-transition-dispose" category="workflow" icon="">disposition-transition-dispose</action>
+    <guard>
+      <guard-role>Records Manager</guard-role>
```
Comment (lukasgraf) [inline opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml]: Not new in this PR, but shouldn't only the Records Manager be able to trigger the `disposition-transition-dispose` transition (according to the diagram)?

File: opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      <guard-role>Archivist</guard-role>
+      <guard-role>Manager</guard-role>
+    </guard>
+  </transition>
+  <transition transition_id="disposition-transition-archive" title="" new_state="disposition-state-archived" trigger="USER" before_script="" after_script="">
+    <action url="%(content_url)s/content_status_modify?workflow_action=disposition-transition-archive" category="workflow" icon="">disposition-transition-archive</action>
+    <guard>
+      <guard-role>Records Manager</guard-role>
```
Comment (lukasgraf) [inline opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml]: Not new in this PR, but shouldn't only the Archivist be able to trigger the `disposition-transition-archive` transition (according to the diagram)?

Comment (phgross) [inline opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml]: Thats true, I've added a separate Issue #2408

File: opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml (hunk 1 above)
Comment (phgross) [inline opengever/disposition/upgrades/20161206181150_add_refuse_transition_to_disposition_workflow/workflows/opengever_disposition_workflow/definition.xml]: Thats true, I've added a separate Issue #2408

**Pull request merged**